packages/cli/lib/configs.js

```diff
@@ -0,0 +1,53 @@
+const request = require('request-promise-native');
```

Comment:
I think it makes sense to move this file into `cli-lib` and have it live under `api`. Maybe name it something like `github.js` since it is being used to make HTTTP API request to GitHub? If we do keep in `packages/cli`, we'll need to add an explicit dep on `request-promise-native`.

Reply:
Yeah, agree. The reason we put thi to cli is that this logic is very specific (downloading config file and parsing it to json), not sure, whether it will be useful somewhere else except using it for our sample apps. But agree, it's not something that directly related to cli logic. Maybe it'd be better to move to cli-lib, since we don't want to add any unnecessary deps to cli

Reply:
@gcorne maybe it will be ok to move these methods to projects.js, since they do similar logic to methods in this module (interact with github api, do something with downloaded data and return it) 
Or, here is a better idea, we can actually create something like github.js, and move methods that interact with github api to this module (fetchReleaseData for example, since currently projects.js has some mixed logic, some methods are fetching data, some methods handle the data locally)